api/cl_ext_cxx_for_opencl.asciidoc

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018-2024 The Khronos Group Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+include::{generated}/meta/{refprefix}cl_ext_cxx_for_opencl.txt[]
+
+=== Other Extension Metadata
+
+*Last Modified Date*::
+    2020-08-25
+*IP Status*::
+    No known IP claims.
+*Contributors*::
+  - Kevin Petit, Arm Ltd. +
+  - Sven Van Haastregt, Arm Ltd. +
+  - Anastasia Stulova, Arm Ltd. +
+  - Marco Antognini, Arm Ltd. +
+  - Neil Hickey, Arm Ltd. +
+  - Alastair Murray, Codeplay +
+
+=== Description
+
+This extension adds support for building programs written using the C++ for
+OpenCL kernel language documented in the *OpenCL-Docs* repository
+(https://github.com/KhronosGroup/OpenCL-Docs)
+with stable versions published in releases of the repository.
+
+This extension also enables applications to query the version of the language
+supported by the device compiler.
+
+=== New Enums
+
+  * {cl_device_info_TYPE}
+  ** {CL_DEVICE_CXX_FOR_OPENCL_NUMERIC_VERSION_EXT}
+
+=== New build option
+
+This extension adds support for a new `CLC++` value to be passed to the
+`-cl-std` build option accepted by *clBuildProgram* and *clCompileProgram*.
```

Comment:
I've not double checked how this looks, but since we're integrating with the unified spec, I think this is the best way to reference entry-points
```suggestion
`-cl-std` build option accepted by {clBuildProgram} and {clCompileProgram}.
```

Reply:
Thanks!